qa/checkstyle/checkstyle.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
```

Comment:
![Screenshot from 2019-05-10 18-22-50](https://user-images.githubusercontent.com/15628173/57542050-d20a9e80-7350-11e9-8ba0-49aa51ee6cea.png)
This is the diff ignoring the whitespace changes